juntagrico/templates/job.html

```diff
@@ -197,19 +198,16 @@ <h3>
                                 {% endfor %}
                             </select>
                         </div>
-                        <br/>
                     {% else %}
                         <input type="hidden" name="jobs" value="1"/>
                     {% endif %}
                     {% for extra in job.empty_per_job_extras %}
-                        <input type="checkbox" name="extra{{ extra.extra_type.id }}" value="{{ extra.extra_type.id }}"/>
+                        <input class="pb-4" type="checkbox" name="extra{{ extra.extra_type.id }}" value="{{ extra.extra_type.id }}"/>
```

Comment:
Here as well the br is legit. now it adds space, that wasn't there before.

Reply:
reverted